README.md

```diff
@@ -30,25 +30,68 @@ Goal: Validate that the evaluated pipeline configuration with its expressions ar
 
 ## Why
 
-Data Factory does not support unit testing out of the box. The only way to validate your changes is through manual testing or running e2e tests against a deployed data factory. These tests are great to have, but miss the following benefits that unit tests, like using this unit test framework, provides:
+Data Factory does not support unit testing out of the box. The only way to validate your changes is through manual testing or running e2e tests against a deployed data factory. These tests are great to have, but miss the following benefits that unit tests, like using this unit test framework, provide:
 
-* Shift left with immediate feedback on changes - Evaluate any individual data factory resource (pipelines, activities, triggers, datasets, linkedServices etc..), including (complex) expressions
+* Shift left with immediate feedback on changes - Evaluate any individual data factory resource (pipelines, activities, triggers, datasets, linked services etc..), including (complex) expressions
 * Allows testing individual resources (e.g. activity) for many different input values to cover more scenarios.
-* Less issues in production - due to the fast nature of writing and running unit tests, you will write more tests in less time and therefore have a higher test coverage. This means more confidence in new changes, less risks in breaking existing features (regression tests) and thus far less issues in production.
+* Less issues in production - due to the fast nature of writing and running unit tests, you will write more tests in less time and therefore have a higher test coverage. This means more confidence in new changes, fewer risks in breaking existing features (regression tests), and thus far fewer issues in production.
 
-> Even though Data Factory is UI-driven and writing unit tests might not be in the nature of it. How can you be confident that your changes will work as expected, and existing pipelines will not break, without writing unit tests?
+> Even though Data Factory is UI-driven writing unit tests might not be in the nature of it. How can you be confident that your changes will work as expected, and that existing pipelines will not break, without writing unit tests?
 
 ## Getting started
 
 1. Set up an empty Python project with your favorite testing library
+   In case you want some suggestions about the setup or libraries that can be used. Please use our following setup suggestion: Docs/EnviromnentSetup/Unittest_setup.md
```

Comment:
Could we keep the non-opiniated readme here and then link to a getting-started.md which contains the opiniated getting started? :)

Reply:
Re-opening this conversation as your code example is still in the main readme :)